docs/add-ons/index.md

```diff
@@ -20,6 +20,8 @@ The framework currently provides support for the following add-ons:
 | [Fargate Fluent Bit](../add-ons/fargate-fluent-bit.md) | Adds Fluent Bit support for EKS Fargate |
 | [Karpenter](../add-ons/karpenter.md) | Deploys Karpenter into an EKS cluster. |
 | [Keda](../add-ons/keda.md) | Deploys Keda into an EKS cluster. |
+| [Kube State Metrics](../add-ons/kube-state-metrics.md) | Deploys Kube State Metrics into an EKS cluster. |
+| [Kubernetes Dashboard](../add-ons/kubernetes-dashboard.md) | Deploys Kubernetes Dashboard into an EKS cluster. |
```

Comment:
### `ToDo` 
We'll need to align https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/docs/add-ons with what we have in https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/modules/kubernetes-addons and use the full URL doc link instead of relative in this file

Reply:
Done.